src/Ledger/PParams.lagda

```diff
@@ -129,7 +130,10 @@ The first four groups have the property that every protocol parameter
 is associated to precisely one of these groups. The \SecurityGroup is
 special: a protocol parameter may or may not be in the
 \SecurityGroup. So, each protocol parameter belongs to at least one
-and at most two groups.
+and at most two groups. Note that in \cite{cip1694} there is no
+\SecurityGroup, but there is the concept of security-relevant protocol
+parameters. The difference between these notions is only social, so we
```

Comment:
I'm not sure what is meant by a difference being "social."  Of course, I can take a guess at what it means... maybe that's good enough?

Reply:
Yeah, that's good enough. Basically, we weren't allowed to call it a group in the CIP because the groups have non-technical stuff attached to them.